Week-4/fibonacci.js

```diff
@@ -0,0 +1,28 @@
+let fibonacciSeries = {
+  [Symbol.iterator]() {
+    let series = [];
+    let currentIndex;
```

Comment:
could have initialised currentIndex to 0 here then there wouldnt have been any need for the condition in line 7

Reply:
Fixed.initialized currentIndex to 0